src/shared_modules/utils/tests/registryHelper_test.cpp

```diff
@@ -50,6 +50,69 @@ TEST_F(RegistryUtilsTest, RegistryDWORDNoThrow)
     EXPECT_FALSE(result);
 }
 
+TEST_F(RegistryUtilsTest, RegistryQWORD)
+{
+    HKEY handler;
+    const LPCTSTR subkey { TEXT("WazuhTest") };
+    LPCTSTR value { TEXT("Test") };
+    ULONGLONG data { 0xF00000000000000 };
+    ULONGLONG valueRead { 0 };
+
+    auto result { RegCreateKeyEx(HKEY_CURRENT_USER, subkey, 0, nullptr, REG_OPTION_NON_VOLATILE, KEY_ALL_ACCESS, nullptr, &handler, nullptr) };
+
+    if (result != ERROR_SUCCESS)
+    {
+        EXPECT_EQ(ERROR_SUCCESS, result);
+    }
```

Comment:
Only keep the assertion
```suggestion
    EXPECT_EQ(ERROR_SUCCESS, result);
```